gem/lib/earlgrey/configure_earlgrey.rb

```diff
@@ -13,19 +13,44 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-# global export for unqualified use in Pods file
-def configure_for_earlgrey(*args)
-  EarlGrey.configure_for_earlgrey(*args)
+require 'colored'
+require 'fileutils'
+require 'xcodeproj'
+
+def configure_for_earlgrey
+  raise 'configure_for_earlgrey is deprecated, please remove.'
 end
 
 module EarlGrey
+  XCScheme = Xcodeproj::XCScheme
+  EnvironmentVariable = XCScheme::EnvironmentVariable
+
+  XCSCHEME_EXT = '*.xcscheme'.freeze
```

Comment:
in modern ruby, strings are frozen by adding a magic comment to the top of the file

`# frozen_string_literal: true`

rubocop will even warn you if this comment isn't present

it's preferable to use this approach instead of invoking .freeze manually on every string.